Hive.NET.Core/Components/Bee.cs

```diff
@@ -33,8 +33,12 @@ public Bee()
 
             beeCallback(this);
             //todo refactor to use internal logging to enable log level filtering
-            _logger.LogDebug("{0:G}: bzzzt, nothing happened", DateTime.UtcNow);
+            //_logger.LogDebug("{0:G}: bzzzt, nothing happened", DateTime.UtcNow);
```

Comment:
Can this be uncommented? or it was intentional?

Reply:
Not intentional, commented for testing - cleaner logs.